java/java.editor/src/org/netbeans/modules/java/editor/semantic/ColoringManager.java

```diff
@@ -87,7 +87,7 @@ public final class ColoringManager {
         put("mod-deprecated", DEPRECATED);
         put("mod-undefined", UNDEFINED);
         put("mod-unused", UNUSED);
-        put("mod-keyword", KEYWORD);
+        //put("mod-keyword", KEYWORD);
```

Comment:
Well, not. I was trying to figure out what exactly it is used for. I'm still not sure.

Reply:
According to the semantic highlighter it would be mostly used from the parser info in `module-info.java` files. But right now we do not parse that section of the code (in the semantic highlighter).